.github/workflows/build-release.yaml

```diff
@@ -42,8 +42,60 @@ jobs:
           docker tag defichain-x86_64-pc-linux-gnu:${ver} defi/defichain:${tag}
         done
         docker push defi/defichain:${ver}
+    
+  linux-armhf:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Populate environment
+      run: GIT_VERSION=1 ./make.sh ci-export-vars
+
+    - name: Build and package
+      run: GIT_VERSION=1 TARGET="arm-linux-gnueabihf" ./make.sh docker-release
 
-  windows:
+    - name: Publish artifact - arm-linux-gnueabihf
+      uses: actions/upload-artifact@v3
+      with:
+        name: defichain-${{ env.BUILD_VERSION }}-arm-linux-gnueabihf
+        path: ./build/defichain-${{ env.BUILD_VERSION }}-arm-linux-gnueabihf.tar.gz
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ env.DOCKER_HUB_USER }}
+        password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+    - name: Push to Docker Hub
+      run: |
```

Comment:
Does this not override linux x64 images if this is published?

Reply:
Resolved in latest commit. Reverted back to pushing only linux-x64 built image into registry.